cppcheck_macros.hpp

```diff
@@ -0,0 +1,25 @@
+// Copyright 2024 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef CPPCHECK_MACROS_HPP_
+#define CPPCHECK_MACROS_HPP_
+
+// Empty macro definition (so that cppcheck ignores the macro)
```

Comment:
> (so that cppcheck ignores the macro)

This comment is redundant. You can remove it

Reply:
Thank you for the advice. I have removed the redundant comment.